pages/common/jq.md

```diff
@@ -0,0 +1,27 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`cat {{file}} | jq`
```

Comment:
That's right @agnivade, `jq` pretty-prints by default. You can add the flag `--compact-output` (or just `-c`) to print one json object per line instead. I'll add this note to the page.